templates/confidential-computing/index.html

```diff
@@ -55,164 +68,154 @@ <h2>Benefits of Ubuntu for confidential computing</h2>
           </ol>
+          <div class="row">
+            <div class="col-3 col-medium-3">
+            </div>
+            <div class="col-6 col-medium-3">
+              <hr class="p-rule--muted" />
+              <a href="https://github.com/canonical/tdx">Deploy Intel TDX with Ubuntu 24.04 today&nbsp;&rsaquo;</a>
+            </div>
+          </div>
         </div>
       </div>
     </section>
 
-    <hr class="p-rule--muted is-fixed-width" />
+    <section class="p-section">
+      <hr class="p-rule is-fixed-width" />
+      <div class="row--50-50">
+        <div class="col">
+          <h2>How confidential VMs work</h2>
+        </div>
+        <div class="col u-sv3">
+          <div class="row">
+            <p>
+              Confidential VMs introduce a new trust boundary which only includes the software running within, and the platform’s hardware. All other software outside is no longer part of your trusted computing base. To provide such strong security guarantees, confidential computing relies on two main primitives:
+            </p>
+          </div>
+          <div class="row u-sv3">
+            <div class="p-image__background--3-2 u-align--center u-vertically-center">
+              {{ image(url="https://assets.ubuntu.com/v1/480ebd6e-how-confidential-vms.png",
+                alt="",
+                width="1200",
+                height="800",
+                hi_def=True,
+                loading="auto|lazy"
+                ) | safe
+              }}
+            </div>
+          </div>
+          <div class="row">
+            <ol class="p-list--divided">
+              <li class="p-list__item p-heading-icon__title p-heading--5">Isolation</li>
+              <div class="col-6 col-medium-3 u-sv3">
+                <p>
+                  Confidential computing capable CPUs are equipped with an AES hardware memory encryption engine, which encrypts data when it is written to system memory, and decrypts it when read. The encryption key itself is stored in the hardware root of trust and is never exposed to the platform’s system software.
+                </p>
+              </div>
+              <li class="p-list__item p-heading-icon__title p-heading--5">Remote attestation</li>
+              <div class="col-6 col-medium-3">
+                <p>
+                  When a confidential VM is launched, its integrity is verified and its initial code and data are measured by a hardware root of trust. This ensures they have not been tampered with. The measurement is cryptographically signed and can be attested to a remote verifier
+                </p>
+              </div>
+            </ol>
+          </div>
+        </div>
+      </div>
+    </section>
 
     <section class="p-strip u-no-padding--top">
+      <hr class="p-rule is-fixed-width" />
       <div class="row">
-        <div class="col-6 col-medium-6">
-          <h2>Enterprise-grade confidential VMs on all major public clouds</h2>
+        <div class="col-12 u-sv3">
```

Comment:
We can move this above the row (since it's basically the same as having 2 rows) and wrap it in a `p-section--shallow`
```suggestion
        <div class="u-fixed-width">
```